src/utils/event.rs

```diff
@@ -56,25 +53,10 @@ impl Events {
                 }
             })
         };
-
-        let tick_handle = {
-            tokio::spawn(async move {
-                loop {
-                    if tx.send(Event::Tick).await.is_err() {
-                        break;
-                    }
-                    tokio::time::sleep(config.tick_rate).await;
-                }
-            })
-        };
-        Events {
-            rx,
-            input_handle,
-            tick_handle,
-        }
```

Comment:
I'm wondering why I never removed this in the first place. Thanks for obliterating it!